src/util/notifications_util.js

```diff
@@ -140,6 +144,13 @@ class Notificator {
         //we can remove the currently processed persistent file
         return true;
     }
+
+    parse_connect_file(connect_filename) {
+        const connect_str = fs.readFileSync(path.join(this.connect_files_dir, connect_filename), 'utf-8');
```

Comment:
when interacting with config files in NC, we should read/write the file using config_fs functionalities
I would try use config_fs.get_config_data() and if it's not exactly what you need, we shall implement it in config_fs.

Reply:
For nsfs, config_fs will be used to load the conf file.
Otherwise (for containerized), same file api is used.